src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age = null,
```

Comment:
by default it will be `undefined`, so you can use it in same manner for conditions you need
```suggestion
    age,
    ...
    age !== undefined
```